pbft/pbft.cpp

```diff
@@ -541,6 +603,16 @@ pbft::wrap_message(const pbft_msg& msg, const std::string& /*debug_info*/)
     return result.SerializeAsString();
 }
 
+bzn::encoded_message
+pbft::wrap_message(const pbft_membership_msg& msg, const std::string& /*debug_info*/) const
+{
```

Comment:
this should return a bzn_envelope; node will not add a signature to an already-serialized message